hack/make/.ensure-images

```diff
@@ -0,0 +1,9 @@
+for image in `docker images | awk '{print $1}'`; do
+if ( [ -z "${image##ppc64le/busybox}" ] || [ -z "${image##s390x/busybox}" ] ); then
+	docker tag $image busybox:latest
+	docker rmi $image
+elif ( [ -z "${image##ppc64le/hello-world}" ] || [ -z "${image##s390x/hello-world}" ] ); then
+	docker tag $image:frozen hello-world:frozen
+	docker rmi $image:frozen
+fi
+done
```

Comment:
We probably not need this once multi-arch support is in place. Hope this solution suffice until then. I prefer not to code generically for more than busybox, hello-world if you agree.